pkgs/by-name/cs/csh/package.nix

```diff
@@ -0,0 +1,64 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  autoPatchelfHook,
+  libbsd,
+}: let
+  inherit (stdenvNoCC.hostPlatform) system;
+  selectSystem = attrs: attrs.${system} or (throw "Unsupported system: ${system}");
```

Comment:
Needs to include the package name to be easier to trace

Reply:
I'm unsure where you'd want it here in the throw output, to avoid further back and forth could you give a suggestion for a string you'd accept?

Reply:
For future reference `throw "csh: Unsupported``